lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1288,14 +1288,18 @@ def _write_soft_mask_groups(self):
             self.output(*content)
             self.endStream()
 
-    def hatchPattern(self, hatch_style):
+    def hatchPattern(self, hatch_style, forced_alpha):
```

Comment:
Does it make sense to give `forced_alpha` a default value of `False`?  That would preserve the API and behavior for anyone who happens to be using thins directly (which I hope there is not, but we have a lot of users who do a lot of very clever things).

Reply:
on the other hand I am the last person to have touched this to add `hatch_style`, did not do this and we had no complaints so we can probably ignore this comment.

Reply:
Maybe just add a warning in the release notes?